elpd_dashboard.ipynb

```diff
@@ -0,0 +1,811 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    global idatas_cmp
</pre><p>There is a way to remove these globals?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz_dashboard/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>